cephfs/admin/response.go

```diff
@@ -3,140 +3,22 @@
 package admin
 
 import (
-	"encoding/json"
-	"errors"
-	"fmt"
-	"strings"
+	"github.com/ceph/go-ceph/internal/commands"
 )
 
 var (
-	// ErrStatusNotEmpty may be returned if a call should not have a status
-	// string set but one is.
-	ErrStatusNotEmpty = errors.New("response status not empty")
-	// ErrBodyNotEmpty may be returned if a call should have an empty body but
-	// a body value is present.
-	ErrBodyNotEmpty = errors.New("response body not empty")
+	// ErrStatusNotEmpty is an alias for commands.ErrStatusNotEmpty
```

Comment:
Maybe its not needed, originally I wanted users to be able to check for that specific error if needed. It's now an alias because the previous type was exported and taking it away would break api compatibility.